src/db.ts

```diff
@@ -1034,20 +1040,21 @@ export abstract class Database {
   public async recordMeasurementBatched10(values: any[]): Promise<number> {
     const q = this.queries.insertMeasurementBatched10;
     // [runId, trialId, invocation, iteration, critId, value];
+    // come back and tidy up comments
```

Comment:
please do!
Please search through the whole code base for `iteration` and identify any missing bits.